manager-bundle/src/Resources/skeleton/config/security.yml

```diff
@@ -52,6 +52,7 @@ security:
                     - contao.security.logout_handler
                 success_handler: contao.security.logout_success_handler
 
+    # Here for BC considerations
     access_control:
```

Comment:
So what does `access_control` actually do then? I thought for the back end firewall to work properly, the route _must_ start with `/contao`, due to the `access_control` configuration. Doesn't this need to be changed as well?

Reply:
Nope, the firewall works because of the route matcher -- the route matcher matches all routes with scope=backend attribute.
The access_control ist just in case someone forgets to set the attribute in the route.

Reply:
Ah I see, thank you for the clarification :). Need to update that in the docs too ;)

Reply:
However, this "security" measure would no longer work if we the path is reconfigured, right? Can't we use the parameter in this path as well?

Reply:
In security.yml, the container parameters are not available (yet).
I consider changing the backend path as "you better know what you do".

Reply:
This would also only affect custom backend routes that usually still have the /contao prefix in the path until one manually updates to the new config parameter.

Reply:
Sorry I might be unclear. The **firewall** uses the scope to run, but it does not use the scope to protected the routes! If you do not have an `isGranted` check, and the current `access_control` no longer takes place, you will have a backend firewall but access will still be granted!

Reply:
So in the current state of `4.x` there will be no access control check anymore, if the back end path is changed?

Reply:
> If you do not have an `isGranted` check, and the current `access_control` no longer takes place, you will have a backend firewall but access will still be granted!

Ok, I guess that's correct. But I want to add that this is not the case when you extend `BackendCustom` or `BackendMain`.

Reply:
But it would be for custom back end controllers. Currently we only mention the following in the docs:

> If the scope is `backend`, the route is automatically protected by the `contao_backend` firewall.

Reply:
While using a custom route prefix, I don't use `isGranted()` in my custom controllers and get redirected to /contao/login for unauthenticated requests. So there must be a different security mechanism in place already.

Reply:
see https://github.com/contao/contao/pull/3720